.vscode/tasks.json

```diff
@@ -168,6 +168,34 @@
             "build-sample-under-dotnet"
           ]
         },
+        {
+            "label": "run-nativeaot-sample",
+            "type": "shell",
+            "windows": { "command": "samples/NativeAOT/runwithdebugger.ps1", },
```

Comment:
Should these commands have a trailing `,`? JSON doesn't normally like that, and GitHub is highlighting it in red…